tests/sync_tests/test_interfaces.py

```diff
@@ -211,8 +212,10 @@ def test_http_proxy(proxy_server: URL, proxy_mode: str) -> None:
 
 
 
-# This doesn't run with trio, since trio doesn't support local_address.
 def test_http_request_local_address() -> None:
+    if current_async_library() == "trio":
+        pytest.skip("trio does not support local_address.")
+
     with httpcore.SyncConnectionPool(local_address="0.0.0.0") as http:
```

Comment:
Note: as I mentioned in https://github.com/encode/httpcore/pull/134#discussion_r465971408, this test doesn't actually _assert_ anything / verify that `local_address` had any effect. And even after some investigation, I'm not sure there's a way to test that cross-platform and cross-machine. Eg it seems I can only bind on `10.6.2.175` locally. But that's what gets selected when using `0.0.0.0` anyway, so passing it as `local_address` wouldn't prove that `local_address` has any effect either…